prometheus_client/exposition.py

```diff
@@ -30,6 +30,11 @@
 
 PYTHON26_OR_OLDER = sys.version_info < (2, 7)
 
+# Default socket timeout for http requests, such as to a pushgateway.
+# Note: Setting this after import time will not change the default used.
+_REQUEST_TIMEOUT = 30
```

Comment:
It's be simpler to hardcode it, this is more than enough time.

Reply:
Using a (pseudo) const is mostly a hint to use a consistent value for anything doing http requests. Can just use literal everywhere if you'd prefer?

Reply:
I'd prefer a literal.

Reply:
@brian-brazil Done.